File: BackEnd/db/models/training.py
from sqlalchemy import Column, Integer, String, Boolean, Date

from db.database import Base

class Training(Base):
    __tablename__ = "Training"

    id = Column(Integer, primary_key=True, index=True)
    description = Column(String)
    level = Column(Integer)
    initial_date = Column(Date)
    end_date = Column(Date)
    institution = Column(String)
    state = Column(Boolean)

File: FrontEnd/src/dto/Training.ts
import type { TrainingLevel } from "@/enum/trainingLevel";

export class Training {
    id: number | null = null;
    description: string = "";
    level: TrainingLevel | null = null;
    initial_date: Date | null = null;
    end_date: Date | null = null;
    institution: string = '';
    state: boolean = true;
    level_text: string = '';
}